recipes-core/systemd/systemd/10-pelux-ethernet.network

```diff
@@ -0,0 +1,6 @@
+[Match]
+Name=en*
+Name=eth*
```

Comment:
I guess this has been the case on all the devices we are trying to support? I can't remember any other common pattern for the naming of the ETH iface.

Reply:
Yes, the naming is taken from how they are named on the NUC and the Pi. I suppose we can amend this list later on if needed.

Reply:
Good!